packages/App/src/balance/statistic/MoneyFlow.tsx

```diff
@@ -0,0 +1,183 @@
+import React, {FC} from 'react';
+import {View, Text, StyleSheet} from 'react-native';
+import {useSelector} from 'react-redux';
+import {walletItems} from '../../store/selectors/walletItems';
+
+enum TransactionType {
+  income = 'income',
+  expenses = 'expenses',
+}
+
+interface Props {
+  month: number;
+}
+
+const MoneyFlow: FC<Props> = ({month}) => {
+  const trans = useSelector(walletItems);
+
+  const allTransactions = trans.map(it => it.transactions).flat();
```

Comment:
If that is a selector which is reused in every component, why not to move it to `selectors` directory and re-import in every place it's used?